cinnamon-remix/README.md

```diff
@@ -0,0 +1,20 @@
+# Lubuntu's Calamares Settings
```

Comment:
No, this is **your** Calamares settings. :)

Reply:
Lol